psalm.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <directory name="src"/>
+        <directory name="test"/>
+        <ignoreFiles>
+            <directory name="vendor"/>
+        </ignoreFiles>
+    </projectFiles>
+
+    <issueHandlers>
+        <InternalMethod>
+            <errorLevel type="suppress">
+                <directory name="test/Exception/"/>
+            </errorLevel>
+        </InternalMethod>
+        <PropertyNotSetInConstructor>
+            <errorLevel type="suppress">
+                <directory name="test/"/>
+                <referencedProperty name="$backupGlobals"/>
```

Comment:
I think this is fixed in newer phpunit versions

Reply:
Just checked latest phpunit version (9.3.7) and problem with `$backupGlobals` is solved, but we are getting another one with `$backupStaticAttributes`, so not sure if I should keep this condition more generic or fixed for now for `$$backupStaticAttributes` property.